### PR TITLE
Add support for after file

### PR DESCRIPTION
**Pull request by quickliketurtle**
Add a check for ~/.lambo/after and source it if it exists.

Add make-after with boiler plate, and edit-after to match config
options.

Update Readme:
Added make-after and edit-after to commands section.
Added new section for After File with examples.

### CODE REVIEW

File: lambo [hunk 1]
```diff
@@ -21,6 +21,9 @@ showhelp()
     echo "${green}   make-config${reset}                  Generate config file"
     echo "${green}   edit-config${reset}                  Edit config file"
     echo " "
+    echo "${green}   make-after${reset}                  Generate after file"
```
Comment (mattstauffer) [inline lambo]: Looks like the indentation is short by one?

File: readme.md [hunk 2]
```diff
@@ -113,6 +113,18 @@ There are also a few optional behaviors based on the parameters you pass (or def
   lambo edit-config
   ```
 
+- `make-after` creates a after file so you can run additional commands after Lambo finishes
```
Comment (mattstauffer) [inline readme.md]: I'd go with:

```markdown
`make-after` creates an "after" file ...
```

**Comment by quickliketurtle** [comment]
Will make those changes... Question: what's the preferred workflow for requested changes? Do I just create another commit with them, or should I squash them into the current commit? 
Thanks.

**Comment by mattstauffer** [comment]
@quickliketurtle Thanks! New commit is fine :)

**Comment by mattstauffer** [comment]
@quickliketurtle thanks! Great work!!

**Pull request merged**